doc/adrs/152_allows_only_coherents_resizes_on_diagram.adoc

```diff
@@ -0,0 +1,33 @@
+= [ADR-152] Allow only coherents resizes on diagram
+
+== Context
+
+When resizing a node on a diagram it's possible for the user to set the dimension too low, this results in the end of the user interaction visually different than the actual final visual. 
+
+Also, there are issues on the layout algorithm that happen when the node is resized below his minimal size.
+
+For example a list nodes that contains 2 `iconLabel` nodes with 2 different `overflow strategy` : `None and Wrap`. Here the minimum size of the parent node cannot be inferior to the size of the sub node with a label strategy of None.
+
+By no allowing the user to interact with the nodes in a way that would cause an invalid layout (even if we try to ajust it to a correct one after), we can improve the user experience.
+
+It should also be easier to avoid invalids interactions with the diagram in the first place rather than correct them during the layout phase afterwards.
```

Comment:
The issue I was trying to fix could have been resolved by fixing the layout after a resize but then I thought it would be a good opportunity to improve the UX